catroid/res/layout/fragment_back_pack_sounds.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Catroid: An on-device visual programming system for Android devices
+  ~ Copyright (C) 2010-2016 The Catrobat Team
+  ~ (<http://developer.catrobat.org/credits>)
+  ~
+  ~ This program is free software: you can redistribute it and/or modify
+  ~ it under the terms of the GNU Affero General Public License as
+  ~ published by the Free Software Foundation, either version 3 of the
+  ~ License, or (at your option) any later version.
+  ~
+  ~ An additional term exception under section 7 of the GNU Affero
+  ~ General Public License, version 3, is available at
+  ~ http://developer.catrobat.org/license_additional_term
+  ~
+  ~ This program is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  ~ GNU Affero General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Affero General Public License
+  ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/fragment_sound"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent" >
+
+    <RelativeLayout
+        android:id="@android:id/empty"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent" >
+
+        <TextView
+            android:id="@+id/fragment_sound_text_heading"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_centerInParent="true"
+            android:gravity="center"
+            android:text="@string/sounds"
+            android:textColor="@color/actionbar_title_color"
+            android:textSize="80sp" />
+
+        <TextView
+            android:id="@+id/fragment_sound_text_description"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_below="@id/fragment_sound_text_heading"
+            android:gravity="center"
+            android:paddingTop="10dp"
+            android:text="@string/fragment_sound_text_description"
+            android:textColor="@color/actionbar_title_color"
+            android:textSize="30sp" />
+    </RelativeLayout>
+
+    <!-- Sound list -->
+
+    <ListView
+        android:id="@android:id/list"
+        style="@style/ListViewShadowed"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_marginTop="3dp"
+        android:cacheColorHint="#00000000"
+        android:divider="@android:color/transparent" />
+
+</FrameLayout>
```

Comment:
Lots of xml files don't end with a newline. The reason is, because newly created xml resources in Android Studio also don't end with a newline and that's also the reason why only java and gradle files are checked for newlines by Checkstyle ([913ffe5](https://github.com/Catrobat/Catroid/commit/913ffe5ee8a1858a74e29567c4611013ba571979)|[f26c688](https://github.com/Catrobat/Catroid/commit/f26c688f6357f597fb00c4391500b6f9481cfd5b)).

Resource xml files with newline at the end: 111
Resource xml files without a newline at the end: 161

Feel free to specify (or discuss) this issue :wink:.

Note#1: Java files created in Android Studio ends with the newline character.
Note#2: The AS formatter settings for xml files have never been defined. Therefore, the Catrobatbot only checks java files. However, line endings can only be checked by Checkstyle anyway.